### PR TITLE
NPM package for fork

### PR DESCRIPTION
**Pull request by rb-cohen**
- Set up package ready to publish to npm as a fork
- This will allow us to download the npm package from npm, rather than copy source and build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,44 +76,19 @@ Tests
 Contributing
 -----
 Feel free to open issues, make suggestions or send PRs.
```
[the remark this answers is not in the document]
Comment (rb-cohen) [reply README.md:78]: I'm down with it. We can reject things that don't align with our goals and worst case take it out.

File: package.json [hunk 2]
```diff
@@ -35,26 +35,34 @@
...
+    {
+      "name": "Son Le",
+      "email": "son.le@ceros.com"
+    },
+    {
+      "name": "Arron Woods",
+      "email": "arron.woods@ceros.com"
     }
   ],
   "license": "MIT",
```
[the remark this answers is not in the document]
Comment (rb-cohen) [reply package.json:48]: Yeah, I'll pull that from their master.

File: README.md [hunk 3]
```diff
@@ -75,44 +76,19 @@ Tests
...
------
-
-Avraam Mavridis : [@avraamakis](https://twitter.com/avraamakis)
-
-Eric Clemmons : [@ericclemmons](https://twitter.com/ericclemmons)
 
 
 Releases
 --------
-- v0.1.0
```
[the remark this answers is not in the document]
Comment (rb-cohen) [reply README.md:91]: It is missing 0.1.0 - 0.40, so I kinda figured we'd start an AF (after-fork) history.. but you're right, no harm in keeping the olf stuff.

**Comment by rb-cohen** [comment]
@noelbennett changes from feedback in. Decided to pivot to Github packages instead of npm, it'll make authentication much easier, as you can use the same ACL as the repo for package management.

Going to convert the other fork, quill-mention, in to a package too... then write a markup branch to consume them both.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (noelbennett) [reply README.md:81]: Sounds good

**Review by noelbennett — APPROVED** [review]
Looks good 👍

**Pull request merged**